generator/template/file_templates.go

```diff
@@ -36,8 +36,9 @@ type {{structImplName}} struct {
 	{{$field.Name}} {{dialect.PackageName}}.Column{{$field.Type}} {{golangComment .Comment}}
 {{- end}}
 
-	AllColumns     {{dialect.PackageName}}.ColumnList
-	MutableColumns {{dialect.PackageName}}.ColumnList
+	AllColumns               {{dialect.PackageName}}.ColumnList
+	MutableColumns           {{dialect.PackageName}}.ColumnList
+	MutableNonDefaultColumns {{dialect.PackageName}}.ColumnList
```

Comment:
I'd rather go with `DefaultColumns` list. This way we can support two use cases:

```golang
Table.AllColumns.Except(Table.DefaultColumns)
// and
Table.MutableColumns.Except(Table.DefaultColumns)
```

Reply:
That sounds good to me!